tsdb/meta.go

```diff
@@ -1121,9 +1132,9 @@ func expandExprWithValues(expr influxql.Expr, keys []string, tagExprs []tagExpr,
 	return exprs
 }
 
-// seriesIDsAllOrByExpr walks an expressions for matching series IDs
+// SeriesIDsAllOrByExpr walks an expressions for matching series IDs
 // or, if no expressions is given, returns all series IDs for the measurement.
-func (m *Measurement) seriesIDsAllOrByExpr(expr influxql.Expr) (SeriesIDs, error) {
+func (m *Measurement) SeriesIDsAllOrByExpr(expr influxql.Expr) (SeriesIDs, error) {
```

Comment:
This is not goroutine safe as well.  Needs a read-lock since `m.seriesIDs` accessed unprotected.  Not sure what `walkWhereForSeriesIds` accesses off-hand.